chain/client/tests/catching_up.rs

```diff
@@ -854,19 +853,19 @@ mod tests {
                     } = msg
                     {
                         let header = partial_encoded_chunk.header.as_ref().unwrap();
-                        println!(
+                        /*println!(
```

Comment:
I prefer removing tracing instead of commenting out.
It's very rare that it needs to be reintroduced, but makes code more messy.

Reply:
++